index.coffee

```diff
@@ -214,12 +208,13 @@ exports.DockerProgress = class DockerProgress
 
 	# Create a stream that transforms `docker.modem.followProgress` onProgress events to include total progress metrics.
 	pullProgress: (image, onProgress) ->
+		progressRenderer = @renderProgress.bind(this)
```

Comment:
I know, but I would have to use fat arrows a couple of times and thought the above is less confusing (and marginally faster).